styles.css

```diff
@@ -161,6 +160,296 @@
     align-items: center;
     justify-content: center;
   }
+
+  /* My Works */
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 64px 24px;
+    gap: 58px;
+    position: relative;
+    width: 100%;
+    height: 3315px;
+    left: 0;
+    top: 0;
+  }
+
+  .works-heading {
+    width: 100%;
+    height: 97px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .works-heading::after {
+    content: '';
+    display: block;
+    border-radius: 10px;
+    width: 50px;
+    height: 5px;
+    background: #36b37e;
+    align-items: center;
+    margin: 15px auto;
+  }
+
+  .works-container {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    grid-template-columns: 100%;
+    grid-template-rows: 16.666% 16.666% 16.666% 16.666% 16.666% 16.666%;
+  }
+
+  .works-samples {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 474px;
+    max-height: fit-content;
+    list-style: none;
+    margin-bottom: 30px;
+    border-radius: 8px;
+    box-shadow: 0 4px 4px rgba(0, 0, 0, 0.5);
+    border: 1px solid #d0d9d4;
+  }
+
+  .works-samples .screenshot {
+    width: 100%;
+    height: 252px;
+    object-fit: cover;
+    border-radius: 8px;
+  }
```

Comment:
- [ ] Kindly align the image and set the true spacings around it as in the figma design:
yours: 
![image](https://user-images.githubusercontent.com/91486791/235227410-56950536-de33-4397-84ba-62a330fce310.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/235227482-bdddf67d-3fe1-4fdc-b1a2-4175bc8d2881.png)

Reply:
But this will mean I will have to give it a fixed width which I was cautioned about yesterday, So just for clarification, should I add a fixed width or find other ways to implement it?